projects/plugins/jetpack/modules/shortcodes/descript.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Descript.com embed
+ *
+ * Example URL: https://share.descript.com/view/jUxUmel6GyN
+ * Example embed code: <iframe src="https://share.descript.com/embed/jUxUmel6GyN" width="640" height="360" frameborder="0" allowfullscreen></iframe>
+ *
+ * @package automattic/jetpack
+ */
+
+if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+	add_action( 'init', 'jetpack_descript_enable_embeds' );
+} else {
+	jetpack_descript_enable_embeds();
+}
+
+/**
+ * Register descript as oembed provider. Add filter to reverse iframes to shortcode. Register [descript] shortcode.
+ *
+ * @since 10.4
+ */
+function jetpack_descript_enable_embeds() {
+	// Support their oEmbed Endpoint.
+	wp_oembed_add_provider( '#https?://share.descript.com/(?:view|embed)/\w+#i', 'https://api.descript.com/v2/oembed', true );
+
+	// Allow script to be filtered to short code (so direct copy+paste can be done).
+	add_filter( 'pre_kses', 'jetpack_shortcodereverse_descript' );
+
+	// Descript rejects any request with the dnt query param, this removes the dnt from the oembed request.
+	add_filter( 'oembed_fetch_url', 'jetpack_descript_remove_dnt' );
```

Comment:
This is a bit annoying. I wonder if we should opt to not embed, but only display a link when dnt is enabled? This way we would honor dnt at all times, and not just ignore it. I think someone who enabled dnt may prefer that over being tracked without knowing about it?

Reply:
Hello, I'm on the Descript engineering team. Good catch! We'll add support for the `dnt` parameter

Reply:
Hey @pdesantis! That's awesome. I've sent you a support email.

Reply:
@alshakero just following up, the `dnt` parameter is now supported